glad/generator/rust/templates/impl.rs

```diff
@@ -76,8 +100,16 @@ mod storage {
     {{ template_utils.protect(command) }} pub static mut {{ command.name|no_prefix }}: FnPtr = FnPtr { ptr: FnPtr::not_initialized as *const raw::c_void, is_loaded: false };
     {% endfor %}
 }
+{% endif %}
 
-pub fn load<F>(mut loadfn: F) where F: FnMut(&'static str) -> *const raw::c_void {
+pub fn load<F>(mut loadfn: F) {{ '-> functions::Gl' if options.mx }} where F: FnMut(&'static str) -> *const raw::c_void {
```

Comment:
I feel like having to separate loader functions would be more readable:

```
if mx:
  define load() -> GL
else:
  define load()
```

Reply:
I assume you mean just the function signature? If so, fixed in https://github.com/Dav1dde/glad/pull/256/commits/3c1d7934900137cecf580409a64b890e30e97a1c. Think it's definitely more readable now.

Reply:
Yeah the function signature or even the whole function, since you're basically splitting the body into two separate parts anyways.